guidelines/sc/20/visual-presentation.html

```diff
@@ -13,5 +13,7 @@ <h4>Visual Presentation</h4>
      <li>Line spacing (leading) is at least space-and-a-half within paragraphs, and paragraph spacing is at least 1.5 times larger than the line spacing.</li>   
      <li>Text can be resized without assistive technology up to 200 percent in a way that does not require the user to scroll horizontally to read a line of text <a>on a full-screen window</a>.</li>   
   </ul>
+
+  <p class="note">The 'mechanism' for achieving the presentation can be provided by the user-agent, which can manipulate the content on behalf of the user. It is not a requirement for the author to provide the mechanism. Some writing systems have additional or different requirements, or do not customarily use features such as paragraph spacing in which case users might not use some or any of these requirements.</p>
```

Comment:
If I were an author reading this note, I think the ending phrase ("in which case users might not use some or any of these requirements.") would confuse me about what exactly my responsibilities as an author are. Does the fact that users might not use these requirements have any impact what I'm required to do as an author to achieve conformance with the SC?

If our intent is to say that authors are not required to make available features inapplicable to the writing system, I'd rather the note be more explicit about that.

I'm not really thrilled about using an informative note to create what ought to be a normative exception, but I can live with it for a AAA criterion. I'd rather accept being a bit uncomfortable with that if it makes it more clear what author requirements are.

Suggested alternative (incorporating Wilco's suggestion above about dropping the redundant "mechanism" note):

```suggestion
<p class="note">Some writing systems have additional or different requirements, or do not customarily use features such as paragraph spacing. When a specific feature is inapplicable to a writing system, this criterion does not require a mechanism to achieve that feature for content authored in that writing system.</p>
```